packages/api-reference/src/components/Card/Card.vue

```diff
@@ -13,7 +13,9 @@
   background: var(--theme-background-2, var(--default-theme-background-2));
   display: flex;
   flex-direction: column;
-  max-height: calc(50vh - 30px);
+  max-height: calc(
+    ((var(--full-height) - var(--theme-header-height, 0px)) - 60px) / 2
```

Comment:
You should add a comment when you add math like this, also I think you want `--refs-header-height` with no fallback.
```suggestion
    ((var(--full-height) - var(--refs-header-height)) - 60px) / 2
```